fn_ipvoid/fn_ipvoid/util/customize.py

```diff
@@ -0,0 +1,744 @@
+# -*- coding: utf-8 -*-
+
+"""Generate the Resilient customizations required for fn_ipvoid"""
+
+from __future__ import print_function
+from resilient_circuits.util import *
+
+def codegen_reload_data():
+    """Parameters to codegen used to generate the fn_ipvoid package"""
+    reload_params = {"package": u"fn_ipvoid",
+                    "incident_fields": [], 
+                    "action_fields": [], 
+                    "function_params": [u"ip_void_artifact_type", u"ip_void_artifact_value", u"ip_void_request_type"], 
+                    "datatables": [], 
+                    "message_destinations": [u"fn_ipvoid"], 
+                    "functions": [u"fn_ipvoid"], 
+                    "phases": [], 
+                    "automatic_tasks": [], 
+                    "scripts": [], 
+                    "workflows": [u"example_ipvoid_dns_lookup", u"example_ipvoid_domain_blacklist_check", u"example_ipvoid_get_ssl_certificate_information", u"example_ipvoid_threat_log", u"example_ipvoid_verify_email_domain", u"example_ipvoidip_reputation"], 
+                    "actions": [u"Example: IPVOID API Check IP Address", u"Example: IPVOID API Domain name", u"Example: IPVOID API Email Domain Check", u"Example: IPVOID API SSL Certificate Information", u"Example: IPVOID API Threat Log Check", u"Example: IPVOID Domain Blacklist Check"] 
+                    }
+    return reload_params
+
+
+def customization_data(client=None):
+    """Produce any customization definitions (types, fields, message destinations, etc)
+       that should be installed by `resilient-circuits customize`
+    """
+
+    # This import data contains:
+    #   Function inputs:
+    #     ip_void_artifact_type
+    #     ip_void_artifact_value
+    #     ip_void_request_type
+    #   Message Destinations:
+    #     fn_ipvoid
+    #   Functions:
+    #     fn_ipvoid
```

Comment:
Function name should not be the same name as the package name. Function and package name were defined in Trello Ticket: https://trello.com/c/vpRNTxqO/24-sudip-ip-void